src/wieldmesh.h

```diff
@@ -44,6 +44,8 @@ class WieldMeshSceneNode: public scene::ISceneNode
 			v3f wield_scale, ITextureSource *tsrc, u8 num_frames);
 	void setItem(const ItemStack &item, IGameDef *gamedef);
 
+	ItemStack getItem();
```

Comment:
Oh, I don't know – I would like a way to access the _current_ wield item at any time, not just the _next_ one (or am I missing something?)